contracts/casino/include/casino/casino.hpp

```diff
@@ -5,6 +5,8 @@
 #include <eosio/asset.hpp>
 #include <platform/platform.hpp>
 
+using namespace eosio;
```

Comment:
don't expose namespace to global, that potentially can cause name collisions in another projects which include this header